pkg/registry/admissionregistration/validatingadmissionpolicybinding/storage/storage_test.go

```diff
@@ -125,8 +125,11 @@ func validPolicyBinding() *admissionregistration.ValidatingAdmissionPolicyBindin
 		},
 		Spec: admissionregistration.ValidatingAdmissionPolicyBindingSpec{
 			PolicyName: "replicalimit-policy.example.com",
-			ParamRef: &admissionregistration.ParamRef{
-				Name: "param-test",
+			ParamRef: &admissionregistration.BindingParamRef{
+				ParamKind: admissionregistration.BindingParamKindClusterWide,
+				ClusterWide: &admissionregistration.ClusterWideParamRef{
+					Name: "param-test",
+				},
```

Comment:
Add test case for per namespace paramRef? Same for below

Reply:
added